flutter_libs/lib/objectbox_flutter_libs.dart

```diff
@@ -1,2 +1,9 @@
-/// This package only contains platform-specific native libraries for flutter.
-/// All the dart code is in package "objectbox".
+/// This package contains platform-specific native libraries for flutter.
+/// See the actual library implementation in package "objectbox".
+
+import 'dart:io';
+
+import 'package:path_provider/path_provider.dart';
+
+Future<Directory> defaultStoreDirectory() async =>
```

Comment:
Some docs on this would be nice, maybe a code snippet on how this should be used?

Edit: this also changes flutter libs to contain Dart code beyond the module loading stuff. Is that an issue? I vaguely remember something about shipping native code only plugins and a Dart issue about this (can't find it right now).

Reply:
added docs

> ... beyond the module loading stuff. Is that an issue? 
No issue I can think of. It's absolutely normal to have code in flutter plugins. Ours was actually weird that it didn't have any :) 

Also, I've realized it's missing `/objectbox` at the end, as we suggested in examples previously, so added that as well.

Reply:
> Also, I've realized it's missing /objectbox at the end, as we suggested in examples previously, so added that as well.

Whoops, missed that. Good catch!